lambda/handler.go

```diff
@@ -72,6 +94,15 @@ func validateReturns(handler reflect.Type) error {
 	return nil
 }
 
+// SetResponseNonEscapeHTML will NOT coerced JSON strings to valid UTF-8,
+// The angle brackets "<" and ">" will NOT be escaped to "\u003c" and "\u003e"
+// Ampersand "&" is also NOT be escaped to "\u0026".
+func SetResponseNonEscapeHTML(flag bool) {
```

Comment:
Instead of a package global setter, I'd prefer optional values to somehow be configured on the type returned by `NewHandler`. I think this could be done in a backwards compatible way in one of these two approaches:

1. adding a `HandlerOptions` type to pass as varargs to `NewHandler`. eg:
  * `NewHandler(handler interface{}, opts... HandlerOptions) lambda.Handler`. 
2. update NewHandler to return a new struct `*JSONHandler` still implementing `lambda.Handler`, and add this add this configuration as a method for that struct. eg:
  * `func NewHandler(handler interface{}) *JSONHandler`
  * `func (handler *JSONHandler) SetEscapeHTML(on bool)` 

I think I like option 2 better, since it would setup a pattern for mirroring the other json encode/decode options like https://golang.org/pkg/encoding/json/#Encoder.SetEscapeHTML

In-use, this'd look like

``` go
func main() {
        jsonHandler := NewHandler(func () (string, error) {
                return "<html><body>html in json string!</body></html>", nil
        });
        jsonHandler.SetEscapeHTML(false);
        lambda.StartHandler(jsonHandler)
}
```